spring-cloud-stream-app-starters-docs/src/main/asciidoc/overview.adoc

```diff
@@ -492,4 +494,146 @@ Then do this:
 
 This would upload the docker image to the docker hub in your custom repository.
 
+=== General FAQ on Spring Cloud Stream App Starters
+
+In the following sections, you can find a brief faq on various things that we discussed above and a few other infrastructure related topics.
+
+
+. What are Spring Cloud Stream Application Starters? +
+Spring Cloud Stream App Starters are individual repositories hosted under the GitHub organization - spring-cloud-stream-app-starters - on github (http://github.com/spring-cloud-stream-app-starters) +
+Project page: http://cloud.spring.io/spring-cloud-stream-app-starters/
+
+
+. What is the parent for stream app starters? +
+The parent for all app starters is `app-starters-build` which is coming from the core project. https://github.com/spring-cloud-stream-app-starters/core
```

Comment:
Does the doc show the actual mvn xml snippit of `<parent/>` ?  That would be a useful addition.